packages/inline-edit/src/styled/InlineEdit.ts

```diff
@@ -42,3 +44,10 @@ export const ButtonWrapper = styled.div<ICommonProps>`
 export const Button = styled.button<ICommonProps>`
   ${({ theme, appearance, baseAppearance }) => getElementStyles(theme, 'button', appearance, baseAppearance)}
 `;
+
+export const ConfirmIcon = styled.svg<ICommonProps>`
```

Comment:
There was a request to add properties to this component with the ability to pass their own icons. I'd rather do it
but we can discuss

Reply:
added